express/middleware/createValidation.mdware.ts

```diff
@@ -0,0 +1,30 @@
+import express, { Request, Response, NextFunction } from "express";
+import fs from "fs";
+import { IUser } from "../types/IUser";
+function createValidation(
+  req: Request,
+  res: Response<string>,
+  next: NextFunction
+) {
+  let username = req.body.username;
+  let id = parseInt(req.body.id);
+
+  if (!id) {
+    return res.status(400).send("Id is required");
+  }
+  if (!username) {
+    return res.status(400).send("Username is required");
```

Comment:
res.json?

Reply:
I didn't quite understand what you mean. Can you explain, please

Reply:
use express.json([options]) method for jsin api. It will set correct content-type and will wrap answer in json